R/drive_upload.R

```diff
@@ -105,16 +105,26 @@ drive_upload <- function(from = NULL,
     up_id <- proc_res$id
   }
 
-  request <- build_request(
-    endpoint = "drive.files.update.media",
-    params = list(
-      fileId = up_id,
-      uploadType = "media",
-      body = httr::upload_file(path = from,
-                               type = mimeType),
-      fields = "*"
-    )
-  )
+  request <- list(url = httr::modify_url(url = .drive$base_url,
```

Comment:
this is not ideal -- the path for updating a new file is a bit buried in the discovery document, so I haven't pulled it out yet, but I should update that soon. @jennybc the drive.files.update endpoint is a bit tricky - in our previous iteration, I manually created two separate endpoints: drive.files.update.meta and drive.files.update.media. Do you think we should do that again, or is there a more elegant way to handle this?